crates/test/src/go.rs

```diff
@@ -0,0 +1,140 @@
+use crate::{Compile, LanguageMethods, Runner, Verify};
+use anyhow::Result;
+use std::env;
+use std::fs;
+use std::path::{Path, PathBuf};
+use std::process::Command;
+
+pub struct Go;
+
+impl LanguageMethods for Go {
+    fn display(&self) -> &str {
+        "go"
+    }
+
+    fn comment_prefix_for_test_config(&self) -> Option<&str> {
+        Some("//@")
+    }
+
+    fn should_fail_verify(
+        &self,
+        _name: &str,
+        config: &crate::config::WitConfig,
+        _args: &[String],
+    ) -> bool {
+        // TODO: We _do_ support async, but only with a build of Go that has
+        // [this
+        // patch](https://github.com/dicej/go/commit/a1c83220fc9576cdb810e9624366cb998e69b22b).
+        // Once we either publish builds containing that patch or upstream
+        // something equivalent, we can remove the ` || config.async_` here.
```

Comment:
How hard would it be to configure CI to have a Go with this patch? Not worth it?